mozilla-release/browser/components/sessionstore/content/aboutSessionRestore.js

```diff
@@ -169,8 +169,10 @@ function restoreSession() {
 }
 
 function startNewSession() {
-  var prefBranch = Cc["@mozilla.org/preferences-service;1"].getService(Ci.nsIPrefBranch);
-  if (!prefBranch.getBoolPref("browser.startup.restoreTabs"))
+  var prefBranch =
+      Cc["@mozilla.org/preferences-service;1"].getService(Ci.nsIPrefBranch);
+  let addNTP = prefBranch.getBoolPref("browser.startup.addFreshTab");
+  if (!addNTP)
```

Comment:
This will make newly opened windows always load the specified start page, instead of about:blank.